src/lib/forms/MultiSelect.svelte

```diff
@@ -66,6 +71,40 @@
       }
     };
   }
+  function handleKeyDown(event) {
+    const focusedElement = document.activeElement;
+    if (event.key === 'Escape') {
+      if (show) {
+        event.stopPropagation();  // We're handling the escape key, so prevent the default behavior, for example, closing a modal
+        show = false;
+      }
+    }
+    else if (event.key === `Enter` || event.key === ` `) {
+      // If the focused element is our div, prevent the form submission and show the dropdown
+      if (focusedElement?.role === 'listbox') {
+        event.preventDefault(); // prevent enter key from triggering form submission
+        // Show the dropdown when pressing enter/space while in focus.
```

Comment:
@shinokada This check is preventing it from working in the context of the documentation, as the focusedElement is 
```
<iframe class="w-full h-full" title="iframe-code-content"></iframe>
```
I don't think this was strictly necessary but I was trying to be conservative.
I can remove the check, which would simplify the function, and make it work in the docs.
Again I'm normally a python dev so all this is quite new to me, I'd love your advice on the matter.